applicationLibrary/sph/SPHLibrary/SPHCalcDensityFunctor.h

```diff
@@ -70,6 +70,16 @@ class SPHCalcDensityFunctor : public autopas::Functor<Particle, SPHCalcDensityFu
     return flops;
   }
 
+  size_t getNumFLOPs() {
+    AutoPasLog(DEBUG, "SPHCalcDensityFunctor::getNumFLOPs called but is not implemented and will return 0.");
+    return 0;
+  }
```

Comment:
Why not move these Debug log warnings to the base class?

Reply:
Not relevant after c31aa16c